### PR TITLE
suggestion to speed up runs

### PR DESCRIPTION
**Pull request by JenniferShelton**
At the moment the contamination marking and merge_bam_files tasks wait for insert size and coverage QC tasks in the scatter to finish. If these tasks run in a separate scatter they should run at the same time as the contamination tasks and the merge_bam_files task.

### CODE REVIEW

File: mondrian/wdl/analyses/alignment.wdl [hunk 1]
```diff
@@ -105,41 +128,52 @@ workflow AlignmentWorkflow{
                 input_bam = merge_sams.output_bam,
                 singularity_dir = singularity_dir
         }
-
-        call picard.CollectWgsMetrics  as wgs_metrics{
+        
+        String cellIds = cellid
+    }
+    
+    scatter (cellinfo in fastq_files) {
```
Comment (amcpherson) [inline mondrian/wdl/analyses/alignment.wdl:135]: @JenniferShelton , couldnt you use zip here?  As in:

```
scatter (cellinfo, cellbam in zip(fastq_files, markdups.output_bam)) {
```

**Comment by JenniferShelton** [comment]
yeah you can make this simpler because you are iterating through the same list twice. I write this like this because as_map is not available in the WDL versions cromwell supports and I am not always going through identical lists. But feel free to swap in anything that takes advantage of the fact that the list is the same in both scatters.

**Comment by diljotgrewal** [comment]
@JenniferShelton 
There is a dependency between the merge_bam_files task and collect_metrics task which in turn depends on all of the metrics calculation tasks.  So I dont think this will change the execution order at all. I'm going to push a fix to remove the dependency in main branch later today.

**Comment by JenniferShelton** [comment]
Yeah @diljotgrewal, if you simplify the input to the tasks that use contamination QC to filter splitting the scatter should allow the picard QC and the merge to start when all the mark duplicate tasks are complete.

**Comment by diljotgrewal** [comment]
fixed in https://github.com/mondrian-scwgs/mondrian/issues/22

scatter does not fix this issue because WF runs the following steps:

- picard wgs metrics generates total read count
- the output from wgs metrics is parsed in collect metrics which in turn depends on other picard metrics collection tasks
- output from collect metrics is then merged with fastqscreen metrics
- this output is then input into mergebamfiles. 

hence there's a dependency in the DAG between merge and all metrics tasks



the fix does the following:
- adds total_reads to fastqscreen summary 
- contamination status is added onto fastqscreen summary file which is now possible since we have read count in this file

this removes the dependency between metrics tasks and the merge, allowing it to start earlier

**Pull request closed without merging**